docs/shared/core-tutorial/06-summary.md

```diff
@@ -0,0 +1,20 @@
+# Core Nx Tutorial - Step 6: Summary
```

Comment:
I'd also mention: add-nx-to-monorepo here saying something like: we've created a new workspace from scratch but you can also add nx to an existing workspace by running ...